### DISCUSSION TITLE
Token authentication requirements for managing organisations in github?

### DISCUSSION THREAD

**Discussion by lfmcmillan** [community/community#23414] [New to GitHub]
<p>OK, so I’ve been notified of the fact that password-based access to Git is ending:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/">
  <header class="source">
      <img src="https://github.blog/wp-content/uploads/2019/01/cropped-github-favicon-512.png?fit=32%2C32" class="site-icon" width="32" height="32">

      <a href="https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/" target="_blank" rel="noopener nofollow ugc" title="08:28PM - 15 December 2020">The GitHub Blog – 15 Dec 20</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://github.blog/wp-content/uploads/2019/09/security-1200-630.png?fit=1200%2C630" class="thumbnail" width="690" height="362"></div>

<h3><a href="https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/" target="_blank" rel="noopener nofollow ugc">Token authentication requirements for Git operations | The GitHub Blog</a></h3>

  <p>Beginning August 13th, 2021, we will no longer accept account passwords when authenticating Git operations on GitHub.com and will instead require token-based authentication (for example, a personal access, OAuth, or GitHub App installation token) for...</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>but what’s happening regarding managing organisations and members’ access rights, etc.? Currently if I want to change stuff in the github organisations I own, Github prompts me for a password.</p>
<p>What’s going to happen in future?</p>
<p>This doesn’t seem to be covered by any of the details on how to set up personal access tokens, because e.g. this page:</p>
<p><a href="https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token" class="onebox" target="_blank" rel="noopener nofollow ugc">https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token</a></p>
<p>doesn’t talk about managing stuff in Github via the browser, it appears to be just about Github access via the command line.</p>
<p>Thanks<br>
Louise</p>

**Comment by shrink** [comment, marked as answer]
<p>The language used in these blog posts is a little confusing. A “Git operation” refers to an activity performed against <a href="http://github.com" rel="noopener nofollow ugc">github.com</a> <em>as a git server</em> and not an activity performed on the website. For example, <code>git push</code> is an authenticated Git operation: today you can <code>git push</code> to <a href="http://github.com" rel="noopener nofollow ugc">github.com</a> with a GitHub username and password but from August 2021 you will need to use a username and <strong>token</strong>.</p>
<p>The GitHub website already takes advantage of improved security because GitHub control it end to end, so the changes described in these blog posts will not impact the GitHub website, only the git server and the GitHub API. Hopefully future announcements from GitHub will mention this nuance :slight_smile:</p>
<p>So in summary, this does not have any impact on managing repository access rights etc. – you will continue to use your password as-is.</p>

**Comment by lfmcmillan** [comment]
<p>Great! Thanks very much for a comprehensive and comprehensible answer.</p>
<p>Louise</p>
